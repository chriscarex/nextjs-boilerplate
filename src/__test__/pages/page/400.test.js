import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Error400 from '../../../pages/page/400';

const setup = () => {
  const component = shallow(<Error400 />);

  return {
    component
  };
};

describe('400 page', () => {
  it('should match snapshot', () => {
    const { component } = setup();
    expect(toJson(component)).toMatchSnapshot();
  });
});
