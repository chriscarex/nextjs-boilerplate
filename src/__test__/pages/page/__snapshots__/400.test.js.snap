// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`400 page should match snapshot 1`] = `
<Fragment>
  <HeadTags
    description="error 400"
    title="Error 400"
  />
  <ErrorPage
    startAgainLink=""
    statusCode={400}
  />
</Fragment>
`;
